### PR TITLE
vmui: add `Raw Query` tab

### PR DESCRIPTION
**Pull request by Loori-R**
### Describe Your Changes

1. **Add new `Raw Query` tab**  
   A new `Raw Query` tab has been added to the [vmui](https://docs.victoriametrics.com/#vmui) interface for displaying raw data. The tab uses the `/api/v1/export` API endpoint. Related issue: [#7024](https://github.com/VictoriaMetrics/VictoriaMetrics/issues/7024)
  
<details>
    <summary>Example:</summary>
    
</details>

2. **Fix rendering of isolated points on the graph**  
   Previously, isolated points (not connected to other points on the left or right) were not visible on the graph. Now, they are rendered correctly.
   
<details>
  <summary>Example:</summary>
  
  | Before Fix | After Fix |
  |----------------|---------------|
  | Before Fix | After Fix |
  
</details>

### Checklist

The following checks are **mandatory**:

- [ ] My change adheres [VictoriaMetrics contributing guidelines](https://docs.victoriametrics.com/contributing/).

### CODE REVIEW

File: app/vmui/packages/vmui/src/pages/RawQueryPage/hooks/useFetchExport.ts [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+    showAllSeries?: boolean;
+  }) => {
+    abortControllerRef.current.abort();
+    abortControllerRef.current = new AbortController();
+    const { signal } = abortControllerRef.current;
+    setIsLoading(true);
+    try {
+      const tempData: MetricBase[] = [];
+      const seriesLimit = showAllSeries ? Infinity : +stateSeriesLimits[displayType] || Infinity;
+      console.log(+stateSeriesLimits[displayType]);
```
Comment (dmitryk-dk) [inline app/vmui/packages/vmui/src/pages/RawQueryPage/hooks/useFetchExport.ts:77]: please remove this logging

**Review by dmitryk-dk — APPROVED** [review]
LGTM! added small comment

File: app/vmui/packages/vmui/src/pages/RawQueryPage/index.tsx [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+        onRunQuery={handleRunQuery}
+        abortFetch={abortFetch}
+        hideButtons={{ traceQuery: true, disableCache: true }}
+        includeFunctions={false}
+      />
+      {showPageDescription && (
+        <Alert variant="info">
+          <div className="vm-explore-metrics-header-description">
+            <p>
+              This page provides a dedicated view for querying and displaying raw data directly from VictoriaMetrics.
```
Comment (hagen1778) [inline app/vmui/packages/vmui/src/pages/RawQueryPage/index.tsx:83]:
```suggestion
              This page provides a dedicated view for querying and displaying raw samples from VictoriaMetrics.
```
Please link `raw samples` to https://docs.victoriametrics.com/keyconcepts/#raw-samples

Comment (hagen1778) [reply app/vmui/packages/vmui/src/pages/RawQueryPage/index.tsx:83]: Please add `It expects only time series selector as a query argument.`
please wrap `time series selector` with https://docs.victoriametrics.com/keyconcepts/#filtering link.

File: app/vmui/packages/vmui/src/pages/RawQueryPage/index.tsx [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+              Users often assume that the <Hyperlink
+                underlined
+                href="https://docs.victoriametrics.com/keyconcepts/#query-data"
+              >Query
+              API</Hyperlink> returns data exactly as stored, but data samples and timestamps may be modified by the API.
+              For more details, see <Hyperlink
+                underlined
+                href="https://docs.victoriametrics.com/single-server-victoriametrics/#how-to-export-data-in-json-line-format"
+              >How
+              to export data in JSON line format</Hyperlink>
```
Comment (hagen1778) [inline app/vmui/packages/vmui/src/pages/RawQueryPage/index.tsx:93]:
```suggestion
```

File: app/vmui/packages/vmui/src/components/Configurators/AdditionalSettings/AdditionalSettings.tsx [hunk 4]
```diff
@@ -67,12 +71,22 @@ const AdditionalSettingsControls: FC<Props & {isMobile?: boolean}> = ({ isMobile
...
+        <Switch
+          label={"Disable cache"}
+          value={nocache}
+          onChange={onChangeCache}
+          fullWidth={isMobile}
+        />
+      )}
+      {!hideButtons?.reduceMemUsage && (
+        <Switch
+          label={"Disable deduplication feature"}
```
Comment (hagen1778) [inline app/vmui/packages/vmui/src/components/Configurators/AdditionalSettings/AdditionalSettings.tsx:84]:
```suggestion
          label={"Disable deduplication"}
```

**Review by hagen1778 — CHANGES_REQUESTED** [review]
It looks great! A few comments though.

And could you please modify the `Export query` modal window by adding the following tip in the bottom:
```
Tip: If confused with the query results, try viewing the raw samples for selected series in `Raw Query` tab.
```

**Comment by Loori-R** [comment]
@hagen1778, PR has been updated.

<details>
  <summary>Updated screenshots</summary>
  
image
image
  
</details>

**Review by hagen1778 — APPROVED** [review]
Great!

**Pull request merged**